restservice/src/test/java/org/itevents/service/transactional/MyBatisEventServiceTest.java

```diff
@@ -169,4 +186,20 @@ private Date getDateInPast(int daysCount) {
         calendar.add(Calendar.DATE, daysCount);
         return calendar.getTime();
     }
+
+    @Test
+    public void shouldReturnLinkToEventSite() throws Exception {
+        Event event = BuilderUtil.buildEventJava();
+        User userGuest = BuilderUtil.buildUserGuest();
+        VisitLog visitLog = VisitLogBuilder.aVisitLog().event(event).user(userGuest).build();
+        String expectedLink = event.getRegLink();
+
+        when(eventService.getEvent(event.getId())).thenReturn(event);
+        when(userService.getAuthorizedUser()).thenReturn(userGuest);
+        doNothing().when(visitLogService).addVisitLog(visitLog);
+
+        String returnedlLink = eventService.redirectToEventSite(event.getId());
+
+        assertEquals(expectedLink, returnedlLink);
+    }
 }
```

Comment:
@IgorMaksymov beacause you tests `eventService.redirectToEventSite()` method that invokes `visitLogService.addVisitLog()` so you have to verify `addVisitLog()`